src/main/java/com/google/sps/data/TestData.java

```diff
@@ -0,0 +1,74 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.sps.util.TestUtil;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/*
+ * Class used for creating mock data for various testing
+ */
+public class TestData {
+  /**
+   * Nested class used to initialize recipes for testing basic recipe name searching
+   */
+  public static final class RecipeNameSearch {
+    // Properties for test recipes in this static class
+    private static final ArrayList<String> TAG_LIST = new ArrayList<String>(
+        Arrays.asList("test", "name", "search"));
+    private static final ArrayList<String> INGRED_LIST = new ArrayList<String>(
+        Arrays.asList("none"));
+    private static final ArrayList<String> STEPS_LIST = new ArrayList<String>(
+        Arrays.asList("none"));
```

Comment:
Why are these not just empty?

Reply:
I was uncertain what would happen if I didn't initialize these values in the backend, once the recipes were retrieved and sent to the front end. That's why I use a helper method to take all the required recipe data and then return an entity. I could have left it as an empty array, too, which is maybe what you're asking about, and I decided against that because if I could create very specific tags for the tests, when I went to delete the entities, I could use the tag search methods I have to remove the test recipes based on name + tag combinations.

Reply:
Yeah, the second one (leaving it as an empty array) is what I was asking about, but if you have a reason to initialize it with "none" then that's fine. Maybe add a comment explaining why you're doing it this way (doesn't have to be part of this PR)